sandstorm-io/sandstorm-website
# Graphics, Styling, Guardfile, Delete PNGs

Completed:
- added and tweaked header graphics for about, blog, how it works, get Sandstorm
- styled subscription form
- added profile pictures and logos, updated bios, also now responsive
- styled features tabs to highlight active
- replaced some pngs with SVGs where applicable
- added gradient to blog header bg, added default bg color for IE
- set all header bgs to the same color (looks way better)
- cleaned up styling for action sections (with buttons and stripe bgs)
- home banner mostly responsive
- other styling tweaks
- (I realize I should create smaller PRs)

Things I'm having trouble with:
- place tents at the bottom of /about contributors section
- get human weblogo under /about corporate sponsors to align vertically in the middle
- remove bottom white padding between /about contributors & footer
- get stripe bgs to appear behind feature graphics (e.g. sandboxing and confinement on /core)
- get rid of small spaces between features subpage navigation tabs (hover over to see)

FYI (Future things not yet done):
- add graphics for: home banner, features banner, features
- style oasis buttons in footer
- review plans/pricing information
- more styling for /get self-host and organizations plans
- add custom font: team social icons, connect with us icons, logo
- review call to actions for each page
- make subscription form work
- update favicon
- review to see if all links go to reasonable places
- test & tweak for mobile screens
- integrate blog with new header
- add garply bio
- code cleanup
